tools/cli/app.go

```diff
@@ -66,9 +66,14 @@ func NewCliApp() *cli.App {
 		},
 		cli.StringFlag{
 			Name:   FlagJWT,
-			Usage:  "optional JWT for authorization",
+			Usage:  "optional JWT for authorization. Either this or --jwt-private-key is needed for jwt authentication",
 			EnvVar: "CADENCE_CLI_JWT",
 		},
+		cli.StringFlag{
+			Name:   FlagJWTPrivateKeyWithAlias,
+			Usage:  "optional private key path to create JWT. Either this or --jwt is needed for jwt authentication. --jwt flag has priority over this one if both provided",
```

Comment:
I've never been quite good with text description @longquanzheng is this decent enough? :laughing:

Reply:
Looks great. Thanks 😊

Reply:
Ohh can you say authorization instead of authentication?

Reply:
This feature doesn’t verify the user identity. The provided jwt token is to verify the access. I think by concept the authentication part (using username password or one login) is done in the OAuth integration by the user.

Reply:
This is my classic typo. Updated :)